packages/common/react-components/src/components/Searchbar/Searchbar.tsx

```diff
@@ -25,7 +24,20 @@ export const Searchbar: FC<{
   return (
     <div className='flex flex-1 flex-col bg-white'>
       <div className={mx('flex flex-1 items-center p-2', border && 'border rounded')}>
-        <Input className='w-full outline-0' spellCheck={false} onChange={handleChange} placeholder='Search...' />
+        <Input
+          label={undefined}
```

Comment:
`label` should never be undefined when required — use the same value as `placeholder` and set `slots: { label: { className: 'sr-only'}}}`

Reply:
I've been meaning to improve this API so it’s not so clunky, so apologies for the inconvenience here.